golang/example.go

```diff
@@ -121,7 +121,7 @@ func main() {
 		return
 	}
 
-	fmt.Printf("Fetcehd %s\n", network.Name)
+	fmt.Printf("Fetched %s\n", network.Name)
```

Comment:
I presume this example should move into the golang API client eventually?

Reply:
yes. doing that now.